src/Debug/debugLayer.ts

```diff
@@ -108,6 +112,32 @@ Object.defineProperty(Scene.prototype, "debugLayer", {
     configurable: true
 });
 
+/**
+ * Enum of inspector action tab
+ */
+export enum DebugLayerTab {
+    /**
+     * Properties tag (default)
+     */
+    PROPERTIES = 0,

```

Comment:
Enums are not upper case:
https://github.com/BabylonJS/Babylon.js/blob/a407a04631d848ac89535d64282af004f0ec2cea/src/Materials/Node/Enums/nodeMaterialSystemValues.ts#L4

Reply:
But not a big deal..I will fix that after the merge ;)

Thanks a ton!